collectd.conf

```diff
@@ -125,4 +125,6 @@ PostCacheChain "PostCache"
 
 # if you have other config, especially for plugins, you can drop them
 # into this directory
+Include "/etc/stackdriver/collectd.d"
+Include "/opt/stackdriver/collectd/etc/collectd.d"
```

Comment:
Let's do this first, so any configs in the new location (which don't load plugins, because that will break) take precedence.

Reply:
Done.